app.js

```diff
@@ -0,0 +1,295 @@
+import './style.css';
+import kbdkeys from './keys';
+
+const states = {
+  firedKey: null,
+  ControlLeft: false,
+  AltLeft: false,
+  ShiftLeft: false,
+  position: null,
+  lang: 'en',
+};
+
+const createEl = (tag, cls, addTo, tagvalue) => {
+  const el = document.createElement(tag);
+  if (cls != null) {
+    el.setAttribute('class', cls);
+  }
+  if (tagvalue != null) {
+    el.innerText = tagvalue;
+  }
+
+  addTo.append(el);
+  return el;
+};
+
+const changeLang = (kbdKeysInst) => {
+  const { lang, ShiftLeft } = states;
+  if (lang === 'en' && ShiftLeft === false) {
+    kbdKeysInst.forEach((keyGroup) => {
+      keyGroup.forEach((key) => {
+        const keyCurrent = document.querySelector(`div[data-action=${key[1]}]`);
+        const keyRu = key[2];
+        keyCurrent.innerText = keyRu;
+      });
+    });
+    states.lang = 'ru';
+  }
+
+  if (lang === 'en' && ShiftLeft === true) {
+    kbdKeysInst.forEach((keyGroup) => {
+      keyGroup.forEach((key) => {
+        const keyCurrent = document.querySelector(`div[data-action=${key[1]}]`);
+        const keyRu = key[3];
+        keyCurrent.innerText = keyRu;
+      });
+    });
+    states.lang = 'ru';
+  }
+
+  if (lang === 'ru' && ShiftLeft === false) {
+    kbdKeysInst.forEach((keyGroup) => {
+      keyGroup.forEach((key) => {
+        const keyCurrent = document.querySelector(`div[data-action=${key[1]}]`);
+        const keyEn = key[4];
+        keyCurrent.innerText = keyEn;
+      });
+    });
+    states.lang = 'en';
+  }
+
+  if (lang === 'ru' && ShiftLeft === true) {
+    kbdKeysInst.forEach((keyGroup) => {
+      keyGroup.forEach((key) => {
+        const keyCurrent = document.querySelector(`div[data-action=${key[1]}]`);
+        const keyEn = key[5];
+        keyCurrent.innerText = keyEn;
+      });
+    });
+    states.lang = 'en';
+  }
+};
+
+const langSwitcher = () => {
+  const { lang } = states;
+  if (lang === 'en') {
+    kbdkeys.forEach((row) => {
+      row.forEach((item) => {
+        const keyCurrent = document.querySelector(`div[data-action=${item[1]}]`);
+        const keyEn = item[2];
```

Comment:
I would prefer to use object instead of array, because we need go back to keys.js  every time  to see what item[0][1][2][3] mean. Using object you can work with names but not numbers